src/tests/ilp-kit.js

```diff
@@ -169,13 +169,18 @@ describe('ILP Kit Test Suite -', function () {
         'Expected an error of type ' + expectedErrorId + ', but was ' + actualErrorId)
     })
 
-    it('Create an user with an invite code', function * () {
+    it('Create a user with an invite code', function * () {
       const config = kitManager.kits[0]
 
       const expectedUser = 'carol'
       const expectedInviteAmount = 222
 
       const code = yield kitManager.createInvite(config, expectedInviteAmount)
+
+      // since the connector was just paid 1000 on creation,
+      // have to wait a bit before paying 222 to this user
+      // due to https://github.com/interledgerjs/five-bells-ledger/issues/402
+      yield sleep(1000)
```

Comment:
good point, created a follow-up ticket for this: https://github.com/interledgerjs/five-bells-integration-test/issues/74